{{cookiecutter.bot_project_name}}/Dockerfile

```diff
@@ -1,43 +1,50 @@
 FROM python:3.9-slim
 
-ENV PYTHONUNBUFFERED 1
-ENV UVICORN_CMD_ARGS ""
-
-ARG NON_ROOT_USER=express_bot
-RUN useradd --create-home ${NON_ROOT_USER}
-
 EXPOSE 8000
-WORKDIR /app
 
-COPY poetry.lock pyproject.toml ./
+ENV PYTHONUNBUFFERED 1
+ENV UVICORN_CMD_ARGS ""
+ENV APP_USER=appuser
 
 ARG CI_JOB_TOKEN=""
 ARG GIT_HOST=""
 ARG GIT_PASSWORD=${CI_JOB_TOKEN}
 ARG GIT_LOGIN="gitlab-ci-token"
 
+
 # Poetry can't read password to download private repos
 RUN echo -e "machine ${GIT_HOST}\nlogin ${GIT_LOGIN}\npassword ${GIT_PASSWORD}" > ~/.netrc && \
     apt-get update && \
     apt-get install -y git sudo && \
-    pip install poetry==1.1.6 --no-cache-dir && \
-    poetry config virtualenvs.create false && \
-    poetry install --no-dev && \
-    echo "${NON_ROOT_USER} ALL = NOPASSWD: /usr/sbin/update-ca-certificates" > /etc/sudoers.d/express_bot && \
-    rm -rf /root/.cache/pypoetry && \
+    echo "${APP_USER} ALL = NOPASSWD: /usr/sbin/update-ca-certificates" > /etc/sudoers.d/update-ca-certificates && \
+    sudo update-ca-certificates && \
+    rm -rf ~/.cache/pypoetry && \
     rm -rf ~/.netrc && \
     apt-get clean autoclean && \
     apt-get autoremove --yes && \
     rm -rf /var/lib/{apt,dpkg,cache,log}/
 
+
+RUN useradd --create-home $APP_USER
+WORKDIR /home/$APP_USER
+USER $APP_USER
+
+ENV VENV_PATH=/home/$APP_USER/.venv/bin
+ENV USER_PATH=/home/$APP_USER/.local/bin
+ENV PATH="$VENV_PATH:$USER_PATH:$PATH"
+
+RUN pip install --user --no-cache-dir poetry==1.1.6 && \
+    poetry config virtualenvs.in-project true
```

Comment:
`poetry config virtualenvs.in-project true` makes poetry to create venv in project directory, while PATH assumes venv in user's home.